stdlib/email/src/main/java/org/ballerinalang/stdlib/email/util/PopConstants.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.email.util;
+
+/**
+ * Constants for POP3-related functions.
+ *
+ * @since 1.1.5
+ */
+public class PopConstants {
+    public static final String DEFAULT_PORT_NUMBER = "995";
+    public static final String PROPS_HOST = "host";
+    public static final String PROPS_PORT = "port";
+    public static final String PROPS_USERNAME = "username";
+    public static final String PROPS_PASSWORD = "password";
+    public static final String PROPS_STORE = "store";
```

Comment:
IMO, these are generic constants for both IMAP and POP protocols. If so, please move it to a different class and refactor the usages.

Reply:
As a single class is used for both POP and IMAP clients, these are considered as part of POP protocol. When they are split these properties can be moved to a generic location.